akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -767,6 +767,7 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
         // tear down the upstream hub part if downstream lane fails
         // lanes are not completed with success by themselves so we don't have to care about onSuccess
         completed.failed.foreach { reason ⇒ hubKillSwitch.abort(reason) }
+        completed.onComplete { x ⇒ println(s"# completed: $x") } // FIXME
 
```

Comment:
I think I got it completely wrong, `sequence` is really one after the other also in case of failure